src/auditwheel/repair.py

```diff
@@ -38,6 +50,7 @@ def repair_wheel(
     update_tags: bool,
     patcher: ElfPatcher,
     strip: bool = False,
+    exclude: Optional[List[str]] = None,
```

Comment:
I would default this to the empty tuple so that we can simplify logic everywhere it's used
```suggestion
    exclude: List[str] = (),
```

Reply:
Thanks!